index.html

```diff
@@ -5,184 +5,277 @@
             </div>
-            <div>
-                <a href="">
-                    <img src="./img/Vector2.png" alt="medium icon">
-                </a>
+            <div class="description-container4">
+                <h2>Tonic</h2>
+                <div class="work-content work-content1">
+                    <p>CANCOPY</p>
+                    <ul>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="work-description">
+                    <p class="p-1">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="categories">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="button">
+                    <button type="button" class="work-button">See Project</button>
+                </div>
             </div>
         </div>
-        <button class="my-resume" type="button">Get my resume</button>
-        <ul class="container">
-            <li class="skills">
-                <h5>Languages</h5>
-                <img src="./img/down-arrow.png" alt="">
-            </li>
-            <li class="list list-item icon">
-                <img src="./img/js.png" alt="">
-                <p>JavaScript</p>
-            </li>
-            <li class="list list-item">
-                <img src="./img/html.png" alt="">
-                <p>HTML</p>
-            </li>
-            <li class="list list-item">
-                <img src="./img/css.png" alt="">
-                <p>CSS</p>
-            </li>
-            <li class="skills line">
-                <h5>Frameworks</h5>
-                <img src="./img/right-arrow.png" alt="">
-            </li>
-            <li class="skills line">
-                <h5>Skills</h5>
-                <img src="./img/right-arrow.png" alt="">
-            </li>
-        </ul>
     </section>
-    <div class="form-container">
-        <form action="https://formspree.io/f/xyyovkvo" method="post">
-            <h2>Contact me</h2>
-            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-            <input type="text" id="Name" name="Name" placeholder="Your name.." minlength="3" maxlength="30" required>
-            <input type="email" id="email" name="email" placeholder="  Your e-mail.." required>
-            <textarea id="message" name="subject" maxlength="500" placeholder="Write your message here" required></textarea>
-            <input type="submit" value="Get in touch">
-        </form>
-        <div class="endline">
+    <section id="about" class="about-myself">
```

Comment:
kindly follow up with the previous reviewer as he requested you change this section to match the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/173181200-0636844c-83c4-4110-9816-ac3b51b51b47.png) | ![image](https://user-images.githubusercontent.com/33268927/173181180-84c962f9-d041-4f6e-88c0-c1fc27ca5c71.png) |

Reply:
@muneebulrehman
Fixed:
- Reduced left section as per the Figma design.